src/legacy/core_plugins/inspector_views/public/requests/_requests.scss

```diff
@@ -4,10 +4,6 @@
   margin-right: $euiSizeS;
 }
 
-.insRequestSelector__menuPanel {
-  width: $euiSizeXL * 2;
```

Comment:
@cchaos Looking at the earlier LESS, that was a min-width (https://github.com/elastic/kibana/commit/f3a4f30b03704101bb6a0f83f2aef4adc6044090#diff-87d86dfef223bfd6db1633faec9e46f3L10) do you still remember why you converted this from `min-width` to `width`?